insights/client/__init__.py

```diff
@@ -237,3 +231,8 @@ def delete_archive(self, path):
             returns (bool): successful archive deletion
         """
         return client.delete_archive(path)
+
+
+def run():
+    c = InsightsClientApi()
+    c.run()
```

Comment:
I don't think we want to instantiate and simply 'run' the Client API. I think we want to have some options/args logic here, like the old client does in 'handle_startup.' Thoughts?